matchbox_socket/src/webrtc_socket/error.rs

```diff
@@ -66,15 +67,10 @@ cfg_if! {
             }
         }
 
-        #[derive(Debug)]
+        #[derive(Debug, Display)]
+        #[display(fmt = "{_0:?}")]
```

Comment:
What's the _0 mean? particularly the `_`

Reply:
Zeroth element of the tuple struct,`_0` because variables cannot start with a numeral